tools/percolator/rollout_nested_collection.xml

```diff
@@ -0,0 +1,52 @@
+<tool id="rollout_nested_collection" name="Rollout nested list" version="3.1">
+    <description>into a single long list</description>
```

Comment:
Aha there is a tool! Thought it was going to be in a future version in javascript. I'll have a try with flatten then! Brb!